x-pack/test/idp-fixture/idp/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM unicon/shibboleth-idp:3.4.2
+
+COPY shibboleth-idp/ /opt/shibboleth-idp/
```

Comment:
Why not pass these in as volumes? 
We could avoid building an image here making it slightly faster.

Reply:
`shibboleth-idp` would work as I'm copying all necessary files, but with the `shib-jetty-base` I'm basically overwriting one file ( `shib-jetty-base/start.d/ssl/ini` ) in the container and I want to use the existing `shib-jetty-base` from the image for the rest. 

I could possibly get the `shib-jetty-base/` out of the image and rewrite the ssl.ini file and pass it as a volume if you think this is preferable.

Reply:
you _can_  mount a single file as a volume, so if it's only the `ini` you care about just append `/ini` to both ends of the volume def